File: dotnet/templates/be-Helpers.cs
// Template: Module helpers
// Location: src/Modules/<ModuleName>/Helpers/<ModuleName>Helpers.cs
// Rules: pure static methods only — no DI, no EF Core, no side effects

namespace YourApp.Modules.ModuleName.Helpers;

public static class ModuleNameHelpers
{
    /// <summary>
    /// Format a ModuleName item for display.
    /// Replace with your actual helper logic.
    /// </summary>
    public static string Format(string id, string name)
        => $"{id}: {name}"; // TODO: implement

    /// <summary>
    /// Filter items by some criteria.
    /// Replace with your actual helper logic.
    /// </summary>
    public static IEnumerable<T> Filter<T>(IEnumerable<T> items, string query)
        => items; // TODO: implement
}

File: python/templates/be-helpers.py
# Template: Module helpers
# Location: src/modules/<module_name>/helpers/<module_name>_helpers.py
# Rules: pure functions only — no FastAPI, no SQLAlchemy, no side effects


def format_module_name_item(item_id: str, name: str) -> str:
    """
    Format a module item for display.
    Replace with your actual helper logic.
    """
    return f"{item_id}: {name}"  # TODO: implement


def filter_module_name_items(items: list, query: str) -> list:
    """
    Filter items by some criteria.
    Replace with your actual helper logic.
    """
    return items  # TODO: implement
